.cirrus.yml

```diff
@@ -54,10 +54,10 @@ task:
       export IS_BITCOIND_25=0
```

Comment:
Should we rename this variable to something like `IS_BITCOIND_LATEST`?

Reply:
It's not the latest we need but anything above 24. Renamed.